Source/WebKit/Shared/mac/AuxiliaryProcessMac.mm

```diff
@@ -504,6 +504,8 @@ static bool tryApplyCachedSandbox(const SandboxInfo& info)
     ASSERT(static_cast<void *>(sandboxDataPtr + profile.size) <= static_cast<void *>(cachedSandboxContents.data() + cachedSandboxContents.size()));
     profile.data = sandboxDataPtr;
 
+    AuxiliaryProcess::setNotifyOptions();
```

Comment:
It seems this should be moved to before `return true;`, otherwise `setNotifyOptions()` can be invoked twice in `static bool applySandbox(...)`.

Reply:
That is a good point. I have changed this to only call `setNotifyOptions` once.